localstack/services/sts/provider.py

```diff
@@ -14,3 +32,31 @@ def get_caller_identity(self, context: RequestContext, **kwargs) -> GetCallerIde
         if "user/moto" in response["Arn"] and "sts" in response["Arn"]:
             response["Arn"] = f"arn:aws:iam::{response['Account']}:root"
         return response
+
+    def assume_role(
+        self,
+        context: RequestContext,
+        role_arn: arnType,
+        role_session_name: roleSessionNameType,
+        policy_arns: policyDescriptorListType = None,
+        policy: unrestrictedSessionPolicyDocumentType = None,
+        duration_seconds: roleDurationSecondsType = None,
+        tags: tagListType = None,
+        transitive_tag_keys: tagKeyListType = None,
+        external_id: externalIdType = None,
+        serial_number: serialNumberType = None,
+        token_code: tokenCodeType = None,
+        source_identity: sourceIdentityType = None,
+        provided_contexts: ProvidedContextsListType = None,
+        **kwargs,
+    ) -> AssumeRoleResponse:
+        response: AssumeRoleResponse = call_moto(context)
+
+        if tags:
+            transformed_tags = {tag["Key"]: tag["Value"] for tag in tags}
+            # we should save it in the store of the role account, not the requester
+            account_id = extract_account_id_from_arn(role_arn)
+            store = sts_stores[account_id]["us-east-1"]
```

Comment:
nit: Might be good to add comment explaining why this is statically set to `us-east-1`. Can be confusing, especially given the effort we're putting into being multi-region everywhere :grin: